packages/cozy-procedures/src/Procedure.jsx

```diff
@@ -1,20 +1,59 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import get from 'lodash/get'
 import { withClient } from 'cozy-client'
-
+import { Document } from 'cozy-doctypes'
+import { Spinner } from 'cozy-ui/transpiled/react/'
 import creditApplicationTemplate from './templates/creditApplicationTemplate'
-
 class Procedure extends React.Component {
   componentDidMount() {
-    this.props.initPersonalData(
+    const {
+      initPersonalData,
+      initDocuments,
+      fetchMyself,
+      fetchDocument,
+      client,
+      setStatusDemarche
+    } = this.props
+    //We init our Document model here to be able to use CozyFile or AdministrativeProcedure models where we want
+    if (!Document.cozyClient) {
+      Document.registerClient(client)
+    }
+    initPersonalData(
       get(creditApplicationTemplate, 'personalData.schema.properties', {})
     )
-    this.props.fetchMyself(this.props.client)
+
+    initDocuments(get(creditApplicationTemplate, 'documents'))
+    //Since init is done, we tell the app we can start to render thing
+    setStatusDemarche({ initiated: true })
```

Comment:
Can we translate `statusDemarche` into english? `procedureStatus` or something like that?